### ISSUE TITLE
Support headless browsers for getBrowserInfo method

### ISSUE THREAD

**Issue by bdwain** [DevExpress/testcafe-browser-tools#150]
Would it be possible to add support for calling `testcafeBrowserTools.getBrowserInfo('chrome:headless')`? I am trying to add a command line flag to chrome because in version 66, trying to autoplay media started failing in our e2es. However, getBrowserInfo fails to recognize chrome:headless and I'd rather not have to add all of the headless browser flags manually.

**Comment by AndreyBelym** [comment]
Hi @bdwain, for Chrome you need `--headless` and `--remote-debugging-port=<port-number>` flags. For Firefox it's just `--headless`. If you have another instances of Chrome/Firefox running, you should terminate them, or start the headless instances with a separate profile (`--user-data-dir=<path> for Chrome and `-profile "<path>" for Firefox). There is another trick for Chrome: you *have to* connect to headless instance with remote debugger, it won't even start loading a page specified in command line arguments until debugger is connected.

`testcafe-browser-tools` was designed to provide a general information about installed browsers and provide some common operations at operating system level. As you see, starting a headless browser is very tricky and browser-specific process, and it's the reason why it requires a dedicated provider in TestCafe.

In short, headless modes are out of scope of the `testcafe-browser-tools` library.

**Comment by bdwain** [comment]
I see. Thanks for explaining.

**Comment by ljowen** [comment]
@bdwain Did you ever find a solution to this? I'm facing the same issue of having to add the --autoplay-policy flag for chrome tests

**Comment by bdwain** [comment]
yep you don't need testcafe-browser-tools. I have a node script to run testcafe for me. it takes the provided browser string (i.e. `chrome` or `firefox` or `chrome:headless`) and does this.

```js
  args.browsers = args.browsers.map(b => {
    if(b.startsWith('chrome')){
      return b + ' --autoplay-policy=no-user-gesture-required'; //note the space
    }
    return b;
  });
```

and args.browsers gets passed to `browsers` method of the runner class (see [here](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html))

**Comment by AndreyBelym** [comment]
@ljowen, you can just append any chrome flag after `chrome:headless`, e.g.
```
testcafe 'chrome:headless --autoplay-policy=no-user-gesture-required' tests.js
```
